apps/james/james.yml

```diff
@@ -5,8 +5,7 @@ repository: https://github.com/james-proxy/james
 category: 'Developer Tools'
 keywords:
     - proxy
-    - debugging-tool
-    - electron
+    - 'debugging tool'
```

Comment:
why was this put in quotes? should we be putting all other multi-part keywords in quotes as well?

Reply:
Force of habit. The real reason this line showed up in the diff was the hyphen removal.

I'm not a yaml expert but from [this backgrounder on whether or not to quote](http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html) it looks like simple strings like the ones in our app metadata are safe to leave unquoted.